internal/domain/bonus/service/service.go

```diff
@@ -0,0 +1,108 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	bonus_transactionsModel "github.com/Azzonya/gophermart/internal/domain/bonusTransactions/model"
```

Comment:
откуда вдруг snake_case появился? Нужно выбрать что-то одно
```suggestion
	bonusTransactionsModel "github.com/Azzonya/gophermart/internal/domain/bonusTransactions/model"
```